zerver/openapi/zulip.yaml

```diff
@@ -6589,11 +6571,38 @@ paths:
         - name: filter_id
           in: path
           description: |
-            The ID of the filter that you want to remove.
+            The ID of the linkifier that you want to remove.
+          schema:
+            type: integer
+          example: 42
+          required: true
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/JsonSuccess"
+    patch:
+      operationId: update_linkifier
+      tags: ["server_and_organizations"]
+      description: |
+        Update [linkifiers](/help/add-a-custom-linkification-filter), regular
```

Comment:
This should be `/help/add-a-custom-linkifier`. This page was renamed in 9223dced3b1f0ca28cf9cf86193e121167ab50ef.